index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Samuel Felice | Full Stack Developer</title>
+</head>
+<body>
+    <header>
+        <span class="logo">My Logo</span> 

```

Comment:
As suggested by the previous reviewer,  it would be better to use an `<a>` tag as it will make the links work better and this will make your code more professional, Kindly Fix it 😀👍